packages/frontend/web-ui/src/home/components/HomeWithAuth.jsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Button, Grid, Link, Typography } from '@mui/material';
+import { Link as RouterLink } from 'react-router-dom';
+import { CornieLayout } from '../../common/layout/CornieLayout';
+import { PendingGame } from '../../game/components/PendingGame';
+import { STATUS_GAME_FULFILLED, useGameStatus } from '../hooks/useGameStatus';
+import { ActiveGame } from '../../game/components/ActiveGame';
+import GamesIcon from '@mui/icons-material/Games';
+
+const GAME_STATUS_NON_STARTED = 'nonStarted';
+const GAME_STATUS_ACTIVE = 'active';
+
+export const HomeWithAuth = () => {
+  const { status, gameList } = useGameStatus();
+  let pendingGames, activeGames;
+
+  if (status === STATUS_GAME_FULFILLED) {
+    pendingGames = gameList.filter(
+      (game) => game.state.status === GAME_STATUS_NON_STARTED,
+    );
+    activeGames = gameList.filter(
+      (game) => game.state.status === GAME_STATUS_ACTIVE,
+    );
```

Comment:
`useGameStatus()` should receive an status so it can be used to fetch active games or non started games. It should allow to fetch games at a certain "page", so we can implement an infinite scroll

Reply:
I'll check it and I'll change it